.github/workflows/test.yml

```diff
@@ -44,18 +44,10 @@ jobs:
     name: "Vulnerability check"
     runs-on: ubuntu-latest
     steps:
-      - name: Check out repository code
-        uses: actions/checkout@v3
-
-      - name: Install Go
-        uses: actions/setup-go@v4
-        with:
-          go-version-file: go.mod
-          check-latest: true
-
       - name: Scan for Vulnerabilities
-        uses: golang/govulncheck-action@v0.2.0
+        uses: golang/govulncheck-action@v1
         with:
+          go-version-file: go.mod
```

Comment:
Remaining warning:
```
[Vulnerability check](https://github.com/Open-CMSIS-Pack/cbuild/actions/runs/6023598641/job/16340657460#step:4:1)
Unexpected input(s) 'go-version-file', valid inputs are ['go-version-input', 'check-latest', 'cache', 'go-package']
```
```suggestion
```

Reply:
The intent was to read the version info from go.mod file. The github action documentation and [implementation ](https://github.com/golang/govulncheck-action/blob/7da72f730e37eeaad891fcff0a532d27ed737cd4/action.yml#L27) mentions that `go-version-file` is a valid input argument. Don't know for some reason it generated a warning.

Fixed it under: https://github.com/Open-CMSIS-Pack/cbuild/actions/runs/6024226883